### ISSUE TITLE
Blur effect

### ISSUE THREAD

**Issue by red-machine95** [aFFekopp/noctis#22]
With update 2021.6.1 the blur effect no longer works

**Comment by aFFekopp** [comment]
I have just upgraded and it's still working for me. Check if it's still uncommented in noctis yaml?

**Comment by red-machine95** [comment]
Yes, the yaml code is uncommented. Before doing the update it was working

**Comment by mochman** [comment]
I had this same problem.  For me, it was because card-mod had a fix for home assistant but the fix hasn't been pushed out to HACS yet.  [Here it the fix](https://github.com/thomasloven/lovelace-card-mod/commit/8e99e2104c7cd9a3c346d3b021400707b7e57b7c), if you want to do update the card yourself.
